### ISSUE TITLE
laravel 5.8 ?

### ISSUE THREAD

**Issue by robov** [BadChoice/handesk#276]
Is it on your roadmap to upgrade your current 5.5 laravel platform to the current 5.8 ?

**Comment by BadChoice** [comment]
Yes, It should be a really easy change, as there are small changes

**Comment by aitriago** [comment]
Hi @robov .
I just updated in my server  to ver. 5.7 with no problem.
But you have to go step-by-step: From 5.5 to 5.6, then to 5.7 (Internet is plenty of documentation about how to do it).
When I tried to setup the 5.8, I got error messages. Actually I'm not working on this upgrade 'cause 5.7 is fine for me ,but for sure I'll do it soon.
